[core]/es_extended/server/classes/player.lua

```diff
@@ -643,15 +643,10 @@ function CreateExtendedPlayer(playerId, identifier, group, accounts, inventory,
 				return print(("[^1ERROR^7] xPlayer.setMeta ^5value^7 should be ^5string^7 as a subIndex!"):format(value))
 			end
 
-            if not self.metadata[index] or type(self.metadata[index]) ~= "table" then
-				self.metadata[index] = { }
-			end
-
+			self.metadata[index] = self.metadata[index] or {} 
```

Comment:
I just merged (1-2 hours), did you test your version? Because I don't think it will work.

Reply:
i tested the statebagchangehandler part that works bcs its replicated to client, but to be sure i will test it and send a video in staff chat.